### ISSUE TITLE
includeInGraphTooltip still being used?

### ISSUE THREAD

**Issue by benloh** [netcreateorg/netcreate-2018#191]
@jdanish is `includeInGraphTooltip` still being used?  I believe you had added this to the template to show/hide tooltips for individual node parameters.  

Are you turning this on and off for certain parameters for different projects?

If it's not being used, I'd like to remove it.  If it *is* being used, I'll add it to the template spec.

**Comment by netcreateorg** [comment]
In theory, we would turn some things on and off. I haven't moved away from whatever I had set the first time I set it up, but the assumption is that the tooltips are handy for quick info (though a hover that is nicely formatted rather than an actual tooltip would be nice one day), but could be too much information / make it too crowded, so we might pick and choose. I think I first implemented this at @kalanicraig's request because of someone using a large network who wanted the info at a glance without having to pick nodes.

tldr; yes it'd be nice, but defaulting to having them all on is fine.

**Comment by benloh** [comment]
Thanks.  I hadn't seen it used in a template before so I wasn't sure.  I'll leave it in.
